www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -64,15 +59,16 @@ Here we are going to set up Helm in your Kubernetes cluster, and then deploy our
 * `helm init --service-account tiller` Installs Tiller, specifying new service account 
 * `helm version` Check and make sure you have the right version of helm running on your client & server side, (as of printing, v 2.8.1) 
 * `kubectl -n kube-system describe deploy/tiller-deploy` Check and see your service account has been correctly set up per instructions
-* `helm repo add habitat-operator https://kinvolk.github.io/habitat-operator/helm/charts/stable/` Adds the Habitat Operator Helm repository to your Helm configuratio
-* `helm dependency update /Users/tdrew/core-plans/nginx/results/nginx-1.11.10-20180220023948/` (or whatever your pwd was -- if it's exactly this we need to talk.) Reads your new nginx-<version>-<timestamp>/requirements.yaml file and sees the dependency on the habitat-operator. Pulls down the chart that describes the Habitat Operator and embeds it in your application’s chart
-* `helm install /Users/tdrew/core-plans/nginx/results/nginx-1.11.10-20180220023948/` Has Helm install your Habitat Helm package on your Minikube cluster
+* `helm repo add habitat-operator https://kinvolk.github.io/habitat-operator/helm/charts/stable/` Adds the Habitat Operator Helm repository to your Helm configuration
+* `helm install habitat-operator/habitat-operator` Installs the Habitat Operator into your Kubernetes cluster using its Helm Chart.
+* `helm dependency update nginx-latest` Reads your new nginx-latest/requirements.yaml file and sees the dependency on the habitat-operator. Pulls down the chart that describes the Habitat Operator and embeds it in your application’s chart
```

Comment:
If you install operator manually (the previous step you added), this step is completely redundant. You should not need the previous step at all actually.